play.py

```diff
@@ -96,11 +96,31 @@ def play_aidungeon_2():
 
     console_print(
         "AI Dungeon 2 will save and use your actions and game to continually improve AI Dungeon."
-        + " If you would like to disable this enter 'nosaving' for any action. This will also turn off the "
+        + " If you would like to disable this enter 'nosaving' at any time. This will also turn off the "
         + "ability to save games."
+        + "Do you want to enter 'nosaving' right now?"
     )
-
-    upload_story = True
+    choice = input("1) Yes, I want to play incognito!\n-) Press enter to skip\n>")
+    if choice == "1":
+        upload_story = False
+ 
+    # "temperature" dictates randomness. A low temperature means that the AI is
+    #  more likely to go with the word that best fits the context, a high
+    #  temperature makes the AI more random and it may chose surprising less fitting words
+    # original 0.4
+    temp = 0.4
+ 
+    # lower top_k is a hard limit of "how many fitting words should I consider",
+    #  i.e. lowering this value also limits the AI in creativity
+    # original 40
+    top_k = 40
+ 
+    console_print("\nBefore we start, would you like to change the default temperature (" + str(temp) + ") and top_k (" + str(top_k) + ") value?\n")
+    choice = input("1) Lower the temp! I want a less random story!\n-) Press enter to skip\n>")
```

Comment:
Most users don't even know what these do. I don't think we should actively prompt the user for these parameters, otherwise people will get confused.

Also, I think the UX for this choice is kinda weird. You ask if they want to change the temperature and top_k, the options say just "lower the temp" (implying that you can only lower the temperature by a predefined amount), and then you prompt for both the temperature and top_k